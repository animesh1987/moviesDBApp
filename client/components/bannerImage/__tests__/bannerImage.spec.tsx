import * as React from 'react';
import { create } from 'react-test-renderer';
/*import { S } from 'enzyme';*/

import { MovieBanner } from '../index';

describe('Movie Banner Test', () => {
  it('renders correctly', () => {
    const imageLink = 'asdac';
    const component = create(<MovieBanner imageUrl={imageLink} />)
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});