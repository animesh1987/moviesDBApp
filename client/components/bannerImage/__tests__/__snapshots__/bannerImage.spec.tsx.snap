// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movie Banner Test renders 1`] = `
<img
  alt="Movie Banner"
  src="asdac"
/>
`;

exports[`Movie Banner Test renders correctly 1`] = `
<img
  alt="Movie Banner"
  src="asdac"
/>
`;
